pkgs/test_core/lib/src/runner/configuration.dart

```diff
@@ -43,6 +43,10 @@ class Configuration {
   bool get help => _help ?? false;
   final bool _help;
 
+  /// Custom HTML template file
+  String get customHtmlTemplatePath => _customHtmlTemplatePath;
```

Comment:
Just make the final variable public here - it already has no public setter.

The other ones have getters because they do some additional logic :)